docs/src/sass/app.scss

```diff
@@ -158,7 +158,7 @@ table thead tr, #content table tbody tr {
 
 /* Header */
 #top-section {
- padding: 1.3rem 0 0.2rem;
+ padding: ($navigation-height + $beta-banner-height) 0 0.2rem;
```

Comment:
I would remove the `$beta-banner-height` here, 50px  padding top looks better from my POV.